okta/config.go

```diff
@@ -17,15 +17,12 @@
 package okta
 
 import (
-	"fmt"
-	"io/ioutil"
-	"os/user"
+	"net/http"
 
-	"github.com/kelseyhightower/envconfig"
-	yaml "gopkg.in/yaml.v2"
+	"github.com/okta/okta-sdk-golang/okta/cache"
 )
 
-type Config struct {
+type config struct {
```

Comment:
why `config` is lowercase while other structs are uppercase?

Reply:
Private vs Public  we don't want config to be used directly